server/planning/events/events.py

```diff
@@ -119,6 +119,10 @@ def _enhance_event_item(self, doc):
         if not doc.get("original_creator"):
             doc.pop("original_creator", None)
 
+        # SDCP-638
+        if not doc.get("language"):
```

Comment:
Almost right. The `on_fetched` and `on_fetched_item` functions modify the item from a `GET` request, which these changes are not stored in the DB.

We want to modify the item on initial creation, which would be the `on_create` function